test/clj_momo/lib/es/document_test.clj

```diff
@@ -310,22 +386,70 @@
                                          "test_index"
                                          "test_mapping"
                                          (query/ids sample-3-ids)
-                                         {:full-hits? true})]
+                                         {:full-hits? true})
+        avg-aggs {:avg_price {:avg {:field :price}}}
+        {data-aggs-1 :data
+         aggs-1 :aggs
+         paging-aggs-1 :paging} (es-doc/query conn
+                                         "test_index"
+                                         "test_mapping"
+                                         {:match_all {}}
+                                         avg-aggs
+                                         {:limit 5})
+
+        stats-aggs {:price_stats {:stats {:field :price}}}
+        {data-aggs-2 :data
+         aggs-2 :aggs} (es-doc/query conn
+                                         "test_index"
+                                         "test_mapping"
+                                         {:match_all {}}
+                                         stats-aggs
+                                         {:limit 0})
+        stats-aggs {:price_stats {:stats {:field :price}}}
+        {data-aggs-3 :data
+         aggs-3 :aggs} (es-doc/query conn
+                                     "test_index"
+                                     "test_mapping"
+                                     (query/ids (map :_id (take 3 sample-docs)))
+                                     stats-aggs
+                                     {:limit 10})]
+
     (is (= (repeat 3 {:foo "bar"})
-           (:data ids-query-result-1))
+           (->> ids-query-result-1
+                :data
+                (map #(select-keys % [:foo]))))
```

Comment:
This test somehow imply the query-result format will be different.
I do not see the previous tests with the previous query-result format. I think if they were modified that we should still keep them to prevent this PR to be a breaking.

Reply:
I modfied the documents, not the query result format for that case:
https://github.com/threatgrid/clj-momo/pull/58/files#diff-6d151f344a372e9d8a6f9f6385e7b238R370
I have added a field, on which I could perform meaningfull aggregations, since documents were only `{:foo "bar"}`.